src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -503,7 +503,7 @@
      * @return A new invalid identity transform
      * @see #createTransform(Extent)
      */
-    Transform createTransform();
+    <E extends Extent> Transform<E> createTransform();
```

Comment:
We'd want the Transform to be generified, so you could do `gameRegistry.<World>createTransform()`